pom.xml

```diff
@@ -50,6 +50,7 @@
         <snakeyaml.version>1.24</snakeyaml.version>
         <spock.version>1.3-groovy-2.5</spock.version>
         <jackson.version>2.9.9</jackson.version>
+        <jaxb-api>2.3.0</jaxb-api>
```

Comment:
and final version 2.3.1

Reply:
I have upgraded the dependency to v2.3.1